src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1152,18 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix(p/, d/, etc) from parameter string if it exists
```

Comment:
good job updating the method comments as well with the actual behaviour of the method.